megago/megago.py

```diff
@@ -35,15 +33,21 @@
 except pkg_resources.DistributionNotFound:
     PROGRAM_VERSION = "undefined_version"
 
-GODAG_FILE_PATH = os.path.join(DATA_DIR, "go-basic.obo")
 
+def is_go_term(candidate):
+    """ Returns true if a given string could be a valid GO-term identifier. This function only checks if the string
+    satisfies the formatting of a GO-term identifier, not if it actually exists in the ontology!
 
-def is_go_term(string):
+    Parameters
+    ----------
+    candidate: A string for which we should verify if it's a valid GO-identifier.
+
+    Returns
+    -------
+    True if the string is indeed a valid identifier, False otherwise.
+    """
     regex = re.compile(r"^go:\d{7}$", re.IGNORECASE)
-    if regex.match(string):
-        return True
-    else:
-        return False
+    return regex.match(candidate)
```

Comment:
```suggestion
    return bool(regex.match(candidate))
```

The regex.match does not return a bool but a match-object or None.